forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -363,3 +364,62 @@ def get():
             )
         except BusinessException as err:
             return err.error, err.status_code
+
+
+@cors_preflight("POST,OPTIONS")
+@API.route("/create-application", methods=["POST", "OPTIONS"])
+class ApplicationCreation(Resource):
+    """Resource for application creation."""
+
+    @staticmethod
+    @auth.require
+    @profiletime
+    def post():
+        """Post a new application using the request body.
+
+        : data: form submission data
+        : formId:- Unique Id for the corresponding form
+        : submissionId:- Unique Id for the submitted form
```

Comment:
this does not make sense. `submissionId` is only available after Formio submission right? how does the client know the submission Id ?